requirements.txt

```diff
@@ -63,21 +64,21 @@ alembic==1.4.2
 amqp==1.4.9
 anyjson==0.3.3
 asn1crypto==1.4.0
-attrs==19.3.0
-awscli==1.18.107
-bcrypt==3.1.7
+attrs==20.1.0
+awscli==1.18.125
+bcrypt==3.2.0
 billiard==3.3.0.23
 bleach==3.1.4
 blinker==1.4
 boto==2.49.0
 boto3==1.12.13
-botocore==1.17.30
+botocore==1.17.48
 certifi==2020.6.20
 chardet==3.0.4
 click==7.1.2
 colorama==0.4.3
 cryptography==3.0
-dnspython==1.16.0
+dnspython==2.0.0
```

Comment:
A bunch of other changes crept in during the `make freeze-requirements` including this one that was a problem before